### DISCUSSION TITLE
Practice mode

### DISCUSSION THREAD

**Discussion by RealCyGuy** [monkeytypegame/monkeytype#701] [Ideas] [answered]
Practice/guest mode:

A mode that doesn't save your scores. Good for when someone else is doing a test, you are practicing slow typing, or you do a custom test that you don't want to mess your history with.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
Can't you just log out whenever someone is typing on your computer? Then log right back in once they're done. That'll also work for the guest mode you want since it won't save your scores. For the custom test you could use a tag so the history goes to the tag instead of your main history.

**Reply by RealCyGuy** [reply]
how do tags work?

**Reply by Miodec** [reply]
You can tag results and then filter them out when viewing your stats. You can create tags in the settings page.

**Reply by RealCyGuy** [reply]
so this wouldn't be a good feature? i could work on it if its wanted

**Reply by frogfrogfrogfrogfrog** [reply]
I don't think Miodec thinks it's necessarily a bad feature. Just one that isn't needed since you can recreate it quite easily. As I said earlier a guest mode is already there whenever you just log out of your account. There's no need to have a completely separate feature for this as Miodec wants to limit the amount of niche things in Monkeytype since there's already so much.

**Reply by Miodec** [reply]
Opening an incognito tab also works as a quick 'dont save my results I wanna mess about' button.
